e2e/process-services-cloud/process-list-cloud-action-menu.e2e.ts

```diff
@@ -0,0 +1,111 @@
+
+        beforeAll(async () => {
+
+            await apiService.login(browser.params.identityAdmin.email, browser.params.identityAdmin.password);
+            identityService = new IdentityService(apiService);
+            groupIdentityService = new GroupIdentityService(apiService);
+            testUser = await identityService.createIdentityUserWithRole(apiService, [identityService.ROLES.APS_USER]);
+            groupInfo = await groupIdentityService.getGroupInfoByGroupName('hr');
+            await identityService.addUserToGroup(testUser.idIdentityService, groupInfo.id);
+
+            await apiService.login(testUser.email, testUser.password);
+            processDefinitionService = new ProcessDefinitionsService(apiService);
+            const processDefinition = await processDefinitionService
+                .getProcessDefinitionByName(resources.ACTIVITI7_APPS.SIMPLE_APP.processes.simpleProcess, simpleApp);
+
+            processInstancesService = new ProcessInstancesService(apiService);
+            editProcess = await processInstancesService.createProcessInstance(processDefinition.entry.key, simpleApp);
+            deleteProcess = await processInstancesService.createProcessInstance(processDefinition.entry.key, simpleApp);
+
+            await settingsPage.setProviderBpmSso(
+                browser.params.config.bpmHost,
+                browser.params.config.oauth2.host,
+                browser.params.config.identityHost);
+            await loginSSOPage.loginSSOIdentityService(testUser.email, testUser.password);
+
+        });
+
+        afterAll(async() => {
+            await apiService.login(browser.params.identityAdmin.email, browser.params.identityAdmin.password);
+            await identityService.deleteIdentityUser(testUser.idIdentityService);
+
+        });
+
+        beforeAll(async () => {
+            await navigationBarPage.navigateToProcessServicesCloudPage();
+            await appListCloudComponent.checkApsContainer();
+            await appListCloudComponent.goToApp(simpleApp);
+            await tasksCloudDemoPage.clickSettingsButton();
+            await tasksCloudDemoPage.enableTestingMode();
+            await tasksCloudDemoPage.enableActionMenu();
+            await tasksCloudDemoPage.enableContextMenu();
+            await tasksCloudDemoPage.addActionIsDisplayed();
+            await tasksCloudDemoPage.addAction('edit');
+            await tasksCloudDemoPage.actionAdded('edit');
+            await tasksCloudDemoPage.addAction('delete');
+            await tasksCloudDemoPage.actionAdded('delete');
+            await tasksCloudDemoPage.clickAppButton();
+            await processCloudDemoPage.clickOnProcessFilters();
+            await processCloudDemoPage.runningProcessesFilter().clickProcessFilter();
+        });
+
+        it('[C315236] Should be able to see and execute custom action menu', async () => {
+            await expect(await processCloudDemoPage.getActiveFilterName()).toBe('Running Processes');
```

Comment:
Do we need to wait for every single line of code?

Reply:
I think so, not sure, what is an alternative to this?